web/src/message_list_view.js

```diff
@@ -107,7 +107,7 @@ function analyze_edit_history(message, last_edit_timestr) {
                 moved = true;
             }
         }
-    } else if (last_edit_timestr !== undefined) {
+    } else if (last_edit_timestamp !== undefined) {
         // When the edit_history is disabled for the organization, we do not receive the edit_history
         // variable in the message object. In this case, we will check if the last_edit_timestr is
```

Comment:
While renaming something (as noted, in it's own commit), please also rename any comments that may refer to it.